packages/content-utils/src/integration/gitPlugin.ts

```
@@ -1,59 +1,143 @@
+		},
+		writeBundle(info, bundle) {
+			if (!info.dir) return;
+			let contentDataEntrypoint: string | undefined;
+			let gitStateEntrypoint: string | undefined;
+			for (const chunk of Object.values(bundle)) {
+				if (chunk.type !== 'chunk') continue;
+				if (chunk.moduleIds.length !== 1) continue;
+
+				switch (chunk.moduleIds[0]) {
+					case RESOLVED_INNER_MODULE_ID:
+						gitStateEntrypoint = joinPath(info.dir, chunk.fileName);
```

Biome (1.9.4)

[error] This case is falling through to the next case.

Add a `break` or `return` statement to the end of this case to prevent fallthrough.

(lint/suspicious/noFallthroughSwitchClause)